### DISCUSSION TITLE
Getting "Whitelabel Error Page" and 404 with the "Getting Started" guide demo

### DISCUSSION THREAD

**Discussion by dventimihasura** [Netflix/dgs-framework#1831] [Q&A] [answered]
Hey folks,

I tried to follow the [Getting Started guide](https://netflix.github.io/dgs/getting-started/) as faithfully as I could, but I'm afraid it didn't quite work.  I'm sure it's something silly I'm overlooking, but when I run the application and try to access the GraphiQL explorer page, I get a "Whitelabel Error Page" and a 404.  FWIW I uploaded the code (what little of it there is) to this GitHub repository:

https://github.com/dventimihasura/netflix-dgs-demo/tree/main

Any thoughts?  Thanks!
Best, David

**Comment by congotej** [comment, marked as answer]
Hi @dventimihasura! Looks like you're on springboot 3.2.3. Can you try bumping up to a more [recent](https://github.com/Netflix/dgs-framework/releases) version of the framework? I know in our docs we have a static `4.9.16` version, but this is quite outdated. I'll go ahead and update that.

**Reply by dventimihasura** [reply]
Thanks @congotej that worked.
